http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java

```diff
@@ -345,6 +345,14 @@ public String clientName() {
          */
         Builder sslProvider(SslProvider sslProvider);
 
+        /**
+         * Set the proxy configuration for this client. The configured proxy will be used to proxy any HTTP request
+         * destined for any host that does not match any of the hosts in {@link ProxyConfiguration#nonProxyHosts() the
```

Comment:
Can you confirm that @link is valid. I've not seen that syntax before.

Reply:
Looks like it is but requires some additional javadoc flags we don't add currently. I'll use a plain `@link` for now.